AboutUs page: make the list numbering continuous #6588

karthikaacharya opened the pull request:
PR Description: Fixes #6588 (Ready for review).

I have added the CSS counters in teammates.css in which it automatically increments the "li" count.

AnaghHegde commented:
@karthikaacharya The contribution guide says like this : 

>PR name: copy-and-paste the relevant issue name and include the issue number as well, e.g. Remove unnecessary System.out.printlns from Java files #3942.

Just copy the issue name and use it as a PR title.

karthikaacharya commented:
Sir, during the second edit I did the same thing as you said, issue name is "AboutUs page: make the list numbering continuous #6588" then also it gave same message.

damithc commented:
@samsontmr can you check the bot response after the first edit? Seems the PR title was correct but the bot responded otherwise.

samsontmr commented:
@damithc Strange, I am unable to reproduce this behavior on my testing repo even after adding a space at the end of the title.

damithc commented:
Let me correct the title and see what happens.

samsontmr commented:
I suspect the first comment was sent when the PR was opened, and the second when @karthikaacharya edited the PR description.

samsontmr commented on src/main/webapp/stylesheets/teammates.css:
```diff
@@ -105,6 +117,8 @@ h2 {
 }
 
+
+
```
Remove these two new lines.

samsontmr commented on .settings/org.eclipse.jdt.core.prefs:
```diff
@@ -1,5 +1,6 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
```
This doesn't seem like something you should be pushing...

damithc commented:
> I suspect the first comment was sent when the PR was opened, and the second when @karthikaacharya  edited the PR description.

The bot comment can be more detailed about what is the problem it is referring to. e.g. whether the problem is in the title or the description

samsontmr commented on src/main/webapp/stylesheets/teammates.css:
```diff
@@ -25,6 +25,18 @@ time, mark, audio, video {
+   list-style-type:none;
+   counter-increment: contributors;
+}
+li:before {
```
Separate these two blocks with a newline

samsontmr commented on src/main/webapp/stylesheets/teammates.css:
```diff
@@ -25,6 +25,18 @@ time, mark, audio, video {
+   counter-reset:contributors;
+}
+
+li{
```
Add a space before`{`.

samsontmr commented:
@damithc Currently, I have disabled the description check so the bot checks only the PR title. I'll enable the description check once I finish writing the code to construct the message is a modular way.

What I suspected happened was:
1. PR created -> payload sent with wrong title.
2. Bot comments.
3. Description modified -> payload sent with wrong title.
4. Bot comments.

samsontmr commented:
@karthikaacharya The fist line of your PR description should be simply: `Fixes #<issue-number>`

karthikaacharya commented:
Ok Sir.

On Sun, Feb 19, 2017 at 7:40 AM, Damith C. Rajapakse <
notifications@github.com> wrote:

> Let me correct the title and see what happens.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/TEAMMATES/teammates/pull/6637#issuecomment-280890145>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AWFq-KnDmVGxDJ-h-jld42jAz3n6S72Oks5rd6SzgaJpZM4MFId9>
> .
>

karthikaacharya commented:
Thank you for your support.

On Sun, Feb 19, 2017 at 8:24 AM, Samson Tan <notifications@github.com>
wrote:

> @karthikaacharya <https://github.com/karthikaacharya> The fist line of
> your PR description should be simply: Fixes #<issue-number>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/TEAMMATES/teammates/pull/6637#issuecomment-280891852>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AWFq-C4NiksLcT8Z52IH-D-diGS8VQDMks5rd675gaJpZM4MFId9>
> .
>

junhaoyap commented:
@karthikaacharya I have set the PR status to ongoing, there are a few changes requested by @samsontmr and there are some conflicts to resolve

karthikaacharya commented:
@junhaoyap I'll do it.

karthikaacharya commented:
yes

karthikaacharya commented:
@samsontmr  I used ctrl+shift+f in eclipse for formatting.

samsontmr commented:
@karthikaacharya Well eclipse doesn't always get it right, and if you did configure your settings, I guess it didn't in this case. You'll just have to really on formatting it yourself in times like these.

karthikaacharya commented:
@samsontmr ok sir.what should I do now?

samsontmr commented:
1. Revert all the unnecessary modifications (this includes the lines where you modified the indentation).
1. Ensure your indentations are only four spaces wide.
1. Revert the line that you committed in `.settings/org.eclipse.jdt.core.prefs`. I believe you should not be committing that.
1. Fix the merge conflicts.

karthikaacharya commented:
Ok.

AnaghHegde commented:
@karthikaacharya Clean your repo. Its messed up.

karthikaacharya commented:
I did not get any conflicts while rebasing 
![fig3](https://cloud.githubusercontent.com/assets/23161592/23275619/3c6007ce-fa2d-11e6-936d-c60dd98350c3.png)
I don't know why it is showing so much conflicts.Please see the attached image.
I got the required output also
![fig5](https://cloud.githubusercontent.com/assets/23161592/23275908/427a7d1e-fa2e-11e6-9c12-9824570455d1.png)
can I send new pull request.

karthikaacharya commented:
This is my first pull request(actually I am new to GitHub) please guide me through this.
I am gonna tell what I did from beginning...
1)I changed teammate.css and about.jsp, then I committed and I did Pull request  for these committed files, there was some indentation problem.
2)I fixed all these in my local machine and came to commit again then I came to know that teammate.css was not there in master branch of TEAMMATES(I think it got deleted).
   but teammate.css was there in my forked repository.Then I thought that my repository was not in sync with main repository. I merged all the commits that was there in teammates.git into my forked repo(I think this was the mistake).
3)Then I checked in my forked repo ,temmate.css was not there then I thought my repo was in sync with teammates.git(master branch).
4)then I made changes in teammtesCommon.css and about.jsp in my local machine and I committed it,after committing it it gave many  conflicts in other files which I not even touched
  .According to my knowledge conflicts occurs only when files I am working on is changed by others. so conflicts should occur in only about.jsp and teammatesCommon.css
  
  Now I don't know what do? if someone guide me it would be a great help.

samsontmr commented:
Is your master branch up to date?

karthikaacharya commented:
yes.

On Fri, Feb 24, 2017 at 9:14 PM, Samson Tan <notifications@github.com>
wrote:

> Is your master branch up to date?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/TEAMMATES/teammates/pull/6637#issuecomment-282323748>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AWFq-LyW44oqmGWZm3KlNT8AnKiNT209ks5rfvr7gaJpZM4MFId9>
> .
>

karthikaacharya commented:
@Samson Tan when I compare my master branch with TEMMATES it shows no
difference.

On Sun, Feb 26, 2017 at 6:59 PM, Karthik R <karthik1996rarsa@gmail.com>
wrote:

> yes.
>
> On Fri, Feb 24, 2017 at 9:14 PM, Samson Tan <notifications@github.com>
> wrote:
>
>> Is your master branch up to date?
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/TEAMMATES/teammates/pull/6637#issuecomment-282323748>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AWFq-LyW44oqmGWZm3KlNT8AnKiNT209ks5rfvr7gaJpZM4MFId9>
>> .
>>
>
>

samsontmr commented:
Since it's updated maybe you can try what I suggested?

karthikaacharya commented:
Ok I'll do it.

karthikaacharya commented:
I will create once again ,since there were so many conflicts I am redoing it everything.
